.github/workflows/neuron-ci.yml

```diff
@@ -37,13 +37,18 @@ jobs:
       INSTALL_DIR: install
       SDK_ROOT: $(xcrun --sdk macosx --show-sdk-path)
       SKIP_WHEELHOUSE_REPAIR: true
+      BUILD_TYPE: Release
+      DESIRED_CMAKE_VERSION: 3.15.0
+      PY_MIN_VERSION: ${{ matrix.config.python_min_version || '3.8' }}
+      PY_MAX_VERSION: ${{ matrix.config.python_max_version || '3.11' }}
+      MUSIC_INSTALL_DIR: /opt/MUSIC
 
     strategy:
       matrix:
         os: [ macOS-11, ubuntu-20.04]
         config:
           - { matrix_eval : "CC=gcc-9 CXX=g++-9",   build_mode: "setuptools"}
-          - { matrix_eval : "CC=gcc-8 CXX=g++-8",   build_mode: "cmake"}
+          - { matrix_eval : "CC=gcc-8 CXX=g++-8",   build_mode: "cmake", music: ON}
```

Comment:
Since we have moved to GCC 9 in https://github.com/neuronsimulator/nrn-build-ci/pull/51, should I replace this with GCC 9 instead?